web/templates/pdf/import/sanctions-licence.html

```diff
@@ -52,7 +52,7 @@
             <p class="address-line row">Department for Business and Trade</p>
             <p class="address-line row">Queensway House, West Precinct</p>
             <p class="address-line row">Billingham, United Kingdom, TS23 2NF</p>  <!-- /PS-IGNORE -->
-            <p class="row min-height-40">Email: {{ ilb_contact_email }}</p>
+            <p class="row min-height-40">Email: {{ sanctions_contact_email }}</p>
```

Comment:
do we want to add `mailto:` here?

```suggestion
            <p class="row min-height-40">Email: <a href="mailto:{{sanctions_contact_email }}">{{ sanctions_contact_email }}</a></p>
```